xknx/remote_value/remote_value.py

```diff
@@ -83,18 +84,16 @@ def unpack_group_addresses(
         self.telegram: Telegram | None = None
         self.after_update_cb: AsyncCallbackType | None = after_update_cb
 
-        if sync_state and self.group_address_state:
-            self.xknx.state_updater.register_remote_value(
-                self, tracker_options=sync_state
-            )
+        # start state updater
+        self.state_updater = StateUpdater(xknx, self, sync_state)
+        self.state_updater.start()
```

Comment:
We can start automatically in `StateUpdater.__init__()` since this isn't optional anyway.